goaway.go

```diff
@@ -45,7 +45,7 @@ func NewProfanityDetector() *ProfanityDetector {
 		profanities:                     DefaultProfanities,
 		falsePositives:                  DefaultFalsePositives,
 		falseNegatives:                  DefaultFalseNegatives,
-		specialCharactersReplacementMap: DefaultSpecialCharacterReplacements,
+		specialCharactersReplacementMap: createIgnoreMap(DefaultIgnoredSpecialCharacters),
```

Comment:
Can you rename `specialCharactersReplacementMap` to `ignoredCharacters` and `DefaultIgnoredSpecialCharacters` to `DefaultIgnoredCharacters`?

`WithSpecialCharacters` should also be renamed to `WithIgnoredCharacters`, and the README should also be updated to reflect the new feature 😅

Reply:
@martinhrvn I see you re-requested my review, but despite 👍ing my comment, but you didn't push any changes 😅

Reply:
@TwiN it must've got stuck in a pipe somewhere 🤦 I pushed now